src/pages/home/index.jsx

```diff
@@ -11,15 +13,13 @@ function HomePage() {
         <Grid item md={6} sm={12}>
           <Typography
             variant="h2"
-            style={{
-              fontSize: "calc(2rem + 6 * ((100vw - 320px) / 680))",
-            }}
+            className={styles.header__home}
           >
-            Exploring possibilities
+           <TypeWriter typing={0.5}>Exploring possibilities</TypeWriter> 
           </Typography>
           <Typography
             variant="h6"
-            style={{ fontWeight: "200", fontStyle: "italic" }}
+            className={styles.headerbottom__home}
           >
             with React Native Elements
```

Comment:
I think it would be good to make it a link which redirects to RNE main repo.

Reply:
yes, it would be a great idea @nlok5923